README.md

```diff
@@ -2,4 +2,4 @@
 
 This project is a basis for a program that controls the _iRobot Create 2_ from another computer, e.g. a Raspberry Pi. 
 
-Generate a runnable jar file and deploy the jar file to the computer controlling the Create 2. Run the program with `java -jar CleverRPiRobot.jar`.
+Clone the repository on the raspberry pi. Run the program with `./gradlew run` on mac or `gradlew run` on PC.
```

Comment:
It's not clear how to run the program. Since the repo is cloned onto the RPi, and the RPi's OS is Linux, one should always use `./gradlew run`.